src/libraries/formDataHandling/functions.ts

```diff
@@ -102,7 +103,7 @@ export const formatAllFieldValues = (
     (acc: Record<string, TFieldFormattedValue>, key) => {
       switch (fields[key].type) {
         case "boolean":
-          acc[key] = values[key] === "true" ? true : false;
```

Comment:
This could use some unit tests

Reply:
We could address this in a separate issue, oder ?